docker/ubuntu-python3/Dockerfile-cuda-9.0

```diff
@@ -22,6 +23,7 @@ RUN apt-get update && apt-get install -y \
     vim \
     ccache \
     graphviz \
+&& pip3 install requests \
```

Comment:
Isn't a --user missing?

Reply:
This one is installed as root; didn't bother checking if it can be installed as user with some environment variable magic. When building docker images on my machine, pip has no issue installing packages as user, but on elk with the same Dockerfile, pip may fail if I don't manually set up environment variables. See for example 4057c0e29 that solved [this build failure](https://gitlab.icp.uni-stuttgart.de/espressomd/docker/-/jobs/181804) on Ubuntu 16.04, while the cuda 10.1 image on Ubuntu 18.04 doesn't have this issue. I'm not sure what's the real cause, though.